autospec/buildreq.py

```diff
@@ -284,6 +284,7 @@ def __init__(self, url):
         self.extra_cmake_openmpi = set()
         self.verbose = False
         self.cargo_bin = False
+        self.pypi_provides = None
```

Comment:
Hmm, should we store this metadata in `provides["pypi"]` instead? I expect that would require more rework though...

Reply:
On second look, I don't see the `"pypi"` key used for either `provides` or `requires`... Then to rephrase my question: Should we store the pypi provides metadata in `provides["python3"]`?

Reply:
We should once we rework provides like I did the requires X(.

Reply:
Okay, I opened #627 to track that work.